R/sql-context.R

```diff
@@ -44,6 +38,9 @@ sql_context__register <- function(name, frame = NULL) {
 sql_context__register_many <- function(...) {
   wrap({
     frames <- list2(...)
+    if (is.null(names(frames)) || any(names(frames) == "")) {
```

Comment:
It seems that `is_named2()` can be used here.

Reply:
Done